### ISSUE TITLE
Easings library

### ISSUE THREAD

**Issue by aaronc** [aaronc/freactive#7]
A stable (possibly 3rd party) easings library. I incorporated some easings from [ominate](https://github.com/danielytics/ominate) - it has some open bug reports - maybe those can be fixed and the easings part can be forked so that it's shared. There's also [tween-clj](https://github.com/gstamp/tween-clj).

**Comment by ul** [comment]
Announced on ML https://github.com/pleasetrythisathome/bardo — looks like interesting.

**Comment by aaronc** [comment]
Yeah, I think it looks pretty good... I'll probably deprecate the easing functions I included with freactive or maybe just include a couple for people to get started with but suggest people use a third-party lib such as bardo.
